helix-term/src/commands/lsp.rs

```diff
@@ -1290,8 +1290,7 @@ pub fn rename_symbol(cx: &mut Context) {
                     match language_server.rename_symbol(doc.identifier(), pos, input.to_string()) {
                         Some(future) => future,
                         None => {
-                            cx.editor
-                                .set_error("Language server does not support symbol renaming");
+                            cx.editor.set_error("Failed to rename symbol");
```

Comment:
This error message is still valid here - that function should only return `None` if the server doesn't support rename requests